### PR TITLE
Fix/skeleton full styles merge conflicts

### PR DESCRIPTION
**Pull request by paulmskim**
Merge `release/B19.14` into bucket branch, resolve merge conflicts

### CODE REVIEW

File: src/Tribe/Main.php [hunk 1]
```diff
@@ -32,7 +32,7 @@ class Tribe__Events__Main {
 		const VENUE_POST_TYPE     = 'tribe_venue';
 		const ORGANIZER_POST_TYPE = 'tribe_organizer';
 
-		const VERSION             = '4.9.9';
+		const VERSION             = '4.9.10';
```
Comment (tr1b0t) [inline src/Tribe/Main.php]: :warning: Equals sign not aligned correctly; expected 1 space but found 13 spaces

<sub>hash: 1eb125f5345e78d2763bd5b0802a3232</sub>

File: src/Tribe/Views/V2/Messages.php [hunk 2]
```diff
@@ -0,0 +1,248 @@
...
+	 * @since TBD
+	 *
+	 * @param string $key The message identification key or slug.
+	 * @param mixed  ...$values A variadic number of arguments that should be used to fill in the message placeholders, if
+	 *                       the message contains `sprintf` compatible placeholders at all.
+	 *
+	 * @return string The human readable message for the specified key, if found, or the key itself.
+	 */
+	public static function for_key( string $key, ...$values ) {
+		$map = [
```
Comment (tr1b0t) [inline src/Tribe/Views/V2/Messages.php]: :no_entry_sign: Expected 1 space between comma and argument "$values"; 0 found

<sub>hash: 30799b265e4a7fd6b12fa427c9d1eabf</sub>

File: src/Tribe/Views/V2/Messages.php [hunk 3]
```diff
@@ -0,0 +1,248 @@
...
+						$highest = (array) reset( $value );
+						// Keep only the first message.
+						$value = [ reset( $highest ) ];
+					}
+				);
+				break;
+			default:
+			case static::RENDER_STRATEGY_LIST:
+			array_walk(
+				$updated_messages,
```
Comment (tr1b0t) [inline src/Tribe/Views/V2/Messages.php]: :no_entry_sign: Line indented incorrectly; expected at least 4 tabs, found 3

<sub>hash: e037d402fadea0e9d64d563f51cd0c58</sub>

File: src/Tribe/Views/V2/Messages.php [hunk 4]
```diff
@@ -0,0 +1,248 @@
...
+				static function ( array &$value, $message_type ) {
+					ksort( $value );
+				}
+			);
+			break;
```
Comment (tr1b0t) [inline src/Tribe/Views/V2/Messages.php]: :no_entry_sign: Line indented incorrectly; expected at least 4 tabs, found 3

<sub>hash: 81386abdf636c837fd33fb62902f6a01</sub>

File: src/Tribe/Views/V2/Messages.php [hunk 5]
```diff
@@ -0,0 +1,248 @@
...
+		}
```
Comment (tr1b0t) [inline src/Tribe/Views/V2/Messages.php]: :no_entry_sign: Terminating statement must be indented to the same level as the CASE body

<sub>hash: 2069a48656281a4ecda8b60d2930d11d</sub>

File: src/Tribe/Views/V2/Template/Excerpt.php [hunk 6]
```diff
@@ -0,0 +1,80 @@
...
+/**
+ * Handles the manipulation of the excerpt.
+ *
+ * @since   4.9.10
+ *
+ * @package Tribe\Events\Views\V2\Template
+ */
+
+namespace Tribe\Events\Views\V2\Template;
+use Tribe__Template as Base_Template;
```
Comment (tr1b0t) [inline src/Tribe/Views/V2/Template/Excerpt.php]: :no_entry_sign: There must be one blank line after the namespace declaration

<sub>hash: 630798932d622b0a555c7b09e2bd10c8</sub>

File: src/Tribe/Views/V2/Template/Excerpt.php [hunk 7]
```diff
@@ -0,0 +1,80 @@
...
+	 * @since 4.9.10
+	 *
+	 * @param int $length The excerpt length.
+	 *
+	 * @return int The excerpt length modified, if necessary.
+	 */
+	public function maybe_filter_excerpt_length( $length ) {
+		$context = tribe_context();
+		$view = $context->get( 'event_display_mode', 'list' );
+
```
Comment (tr1b0t) [inline src/Tribe/Views/V2/Template/Excerpt.php]: :warning: Equals sign not aligned with surrounding assignments; expected 4 spaces but found 1 space

<sub>hash: eb552c6855c1bfde2eebf26682481c26</sub>

File: src/Tribe/Views/V2/Template/Title.php [hunk 8]
```diff
@@ -366,7 +366,7 @@ protected function build_category_title( $title, $cat ) {
 		/**
 		 * Filters the Event Category Archive title.
 		 *
-		 * @since TBD
+		 * @since 4.9.10
```
Comment (tr1b0t) [inline src/Tribe/Views/V2/Template/Title.php]: :no_entry_sign: There must be a single blank line after a tag group

<sub>hash: 664acb465a3f173920a31329050eac59</sub>

File: src/admin-views/aggregator/settings.php [hunk 9]
```diff
@@ -5,7 +5,7 @@
  */
 $internal = array();
 $use_global_settings_phrase = esc_html__( 'Use global import settings', 'the-events-calendar' );
-$post_statuses = get_post_statuses( array() );
+$post_statuses = get_post_statuses();
```
Comment (tr1b0t) [inline src/admin-views/aggregator/settings.php]: :warning: Equals sign not aligned with surrounding assignments; expected 14 spaces but found 1 space

<sub>hash: 17ad3970ecaab2cec118a465303c3247</sub>

**Comment by bordoni** [comment]
Once https://github.com/moderntribe/the-events-calendar/pull/2766 is merged the tests will pass. cc: @lucatume

**Pull request merged**